docs/join2019_submisson.md

```diff
@@ -0,0 +1,29 @@
+#### Session Title
+
+### A Reliable Development Workflow with CI for Looker
```

Comment:
I'm trying to figure out what angle is most sexy for this type of tool. 

What do you think of something like: `Develop LookML Like a Software Engineer: CI for Looker". I don't think the title is likely to make a huge difference, but I think we can probably make it pretty punchy.

Reply:
Or even making it explicitly about the tool (which has a recognisable name)? 

"Introducing Fonz: a CI tool for reliable LookML and warehouse development"

Reply:
I like both of those a lot better. I kept thinking... "How do we _actually_ get people excited about attending a session on CI?"

I think the first one is slightly better. It feels a bit more provocative to me.